tests/perf-system/submitter/submit.cpp

```diff
@@ -237,7 +210,8 @@ int main(int argc, char** argv)
     for (size_t req = 0; req < requests_size; req++)
     {
       gettimeofday(&start[req], NULL);
-      connection->write(raw_reqs[req]);
+      auto request = data_handler.request[req];
+      connection->write({request.data(), request.size()});

```

Comment:
Not sure what write() takes here, but if it's a vector ref, this will result in a temporary vector being allocated, and we definitely do not want that.

Reply:
It's a span, so that's all good.